module/Motion/Rotation.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file   Rotation.cpp
+ * @brief  回頭動作の中間クラス
+ * @author desty505
+ */
+
+#include "Rotation.h"
+
+using namespace std;
+
+Rotation::Rotation(double _targetSpeed, bool _isClockwise)
+  : targetSpeed(_targetSpeed), isClockwise(_isClockwise)
+{
+}
+
+void Rotation::run()
+{
+  // 事前条件を判定する
+  if(runPreconditionJudgement(targetSpeed)) {
+    return;
+  }
+
+  // isClockwiseがtrueなら時計回り，falseなら反時計回り
+  int leftSign = isClockwise ? 1 : -1;
+  int rightSign = isClockwise ? -1 : 1;
+
+  // 呼び出し時の走行距離
+  double leftMileage = Mileage::calculateWheelMileage(Measurer::getLeftCount());
+  double rightMileage = Mileage::calculateWheelMileage(Measurer::getRightCount());
```

Comment:
角度(距離)指定だけでしか使わないから、ここで計算する必要ないと思う。

もしくは、色指定でも回頭した角度を保持したいってことならメンバ変数として保持すべきだと思う。

Reply:
メンバ変数として保持するように書き直したつもりですが、
やり方があっているか分からないので確認いただきたいです。

Reply:
```suggestion
  // 呼び出し時の走行距離
  initLeftMileage = Mileage::calculateWheelMileage(Measurer::getLeftCount());
  initRightMileage = Mileage::calculateWheelMileage(Measurer::getRightCount());
```

①メンバ変数は.cpp側では型定義(double)しなくていい
②命名は、初期の走行距離だとわかるように`init`とか付けてほしい(.h側も要変更)